pkg/cmd/start/main.go

```diff
@@ -81,6 +86,9 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().Int32("metrics-port", 8383, "The metrics port")
 	viper.BindPFlag("metrics-port", cmd.Flags().Lookup("metrics-port"))
 
+	cmd.Flags().Int32("cr-metrics-port", 8686, "The metrics port for Operator and/or Custom Resource based metrics")
```

Comment:
What is the difference between port 8383 and this one?

Reply:
Honestly, I don't know yet. I tried to figure out by reading the docs, but I think I'll understand only when I see the metrics (which I haven't done yet).

Reply:
Ok, I looked into this and it makes sense now: the new endpoint is for metrics about the deployed CRs. I could find only one gauge, like this:

```
opentelemetryservice_info{namespace="default",opentelemetryservice="simplest"} 1
```

The older port (8383) is holding metrics about the operator itself, like:

```
go_goroutines 45
...
rest_client_request_latency_seconds_count{url="https://10.96.0.1:443/api?timeout=32s",verb="GET"} 5
...
workqueue_work_duration_seconds_bucket{name="opentelemetryservice-controller",le="1e-08"} 0
```